packages/libs/react-ui/src/components/Link/Link.stories.tsx

```diff
@@ -1,113 +1,249 @@
+        <br/><br/>
+        To support client side routing make sure to import/use "RouterProvider" from "@kadena/react-ui" see https://react-spectrum.adobe.com/react-aria/routing.html for more info on how to integrate it with NextJS and client side routing.
+        `,
       },
     },
   },
   argTypes: {
-    href: {
-      description:
-        "The href prop that is passed to the anchor or child element. If you're using the 'asChild' prop, you can pass the href to the child element and leave it undefined on the Link element. In times when both are defined, the child element href will be used.",
-      control: {
-        type: 'text',
+    onClick: {
+      action: 'clicked',
+      description: '(deprecated) callback when button is clicked',
+      table: {
+        disable: true,
       },
     },
-    target: {
+    variant: {
+      options: buttonVariants,
       control: {
         type: 'select',
-        options: ['_blank', '_self', '_parent', '_top'],
+      },
+      description: 'button style variant',
+      table: {
+        type: { summary: buttonVariants.join(' | ') },
+        defaultValue: { summary: 'default' },
       },
     },
-    icon: {
-      options: [
-        ...['-'],
-        ...Object.keys(SystemIcon),
-      ] as (keyof typeof SystemIcon)[],
+    color: {
+      options: buttonColors,
       control: {
         type: 'select',
       },
+      description: 'button color variant',
+      table: {
+        type: { summary: buttonColors.join(' | ') },
+        defaultValue: { summary: 'default' },
+      },
     },
-    iconAlign: {
-      description: 'Align icon to left or right',
-      options: ['left', 'right'] as ILinkProps['iconAlign'][],
+    isDisabled: {
+      description: 'only used when rendered as button',
       control: {
-        type: 'radio',
+        type: 'boolean',
       },
-      if: { arg: 'selectIcon', neq: '-' },
     },
-    asChild: {
-      description:
-        "Use this prop when you're using a different Link component and want to pass on styles, icons, and additional props. For example when using next/link in Next.js.",
+    isLoading: {
+      description: 'loading state',
+      control: {
+        type: 'boolean',
+      },
     },
-    block: {
-      description: 'Set to true to make the link a block element.',
+    isCompact: {
+      description: 'compact button style',
+      control: {
+        type: 'boolean',
+      },
     },
   },
 };
 
-export default meta;
-
-type Story = StoryObj<
+type LinkStory = StoryObj<
   {
-    selectIcon: keyof typeof SystemIcon;
+    text: string;
   } & ILinkProps
 >;
 
-export const Primary: Story = {
+export const _Link: LinkStory = {
   name: 'Link',
   args: {
-    href: 'https://kadena.io',
-    target: '_blank',
-    icon: 'Link',
-    iconAlign: 'left',
+    text: 'Click me',
+    variant: 'contained',
```

Comment:
can we change this to undefined or "text" since that will be the default variant for the Link component?

Reply:
@salamaashoush for some reason the Link styles in the default story look different from the others 
<img width="256" alt="image" src="https://github.com/kadena-community/kadena.js/assets/9022549/d8fd6fd8-35f9-431d-8d6f-7a350ab45af5">